pkg/chart/loader/load.go

```diff
@@ -29,6 +29,14 @@ import (
 	"helm.sh/helm/v3/pkg/chart"
 )
 
+// BufferedFile represents a file as a name/value pair.
+//
+// By convention, name is a relative path within the scope of the chart's
+// base directory.
+//
+// DEPRECATED: use chart.File instead
+type BufferedFile = chart.File
```

Comment:
Why have a deprecation in a x.x.0 release?

Reply:
Not technically deprecated. The user can still choose to use either field.

Would a better comment message be

```
// TODO: remove this in Helm 4
```

?

Reply:
Why not remove all together?

Reply:
My guess is that now that we are RC, the time for breaking changes is past

Reply:
SGTM. Will remove.